src/corevideo.cs

```diff
@@ -426,6 +431,55 @@ interface CVPixelBuffer {
 		[Field ("kCVPixelBufferOpenGLTextureCacheCompatibilityKey")]
 		NSString OpenGLTextureCacheCompatibilityKey { get; }
 #endif
+		[Unavailable (PlatformName.MacCatalyst)]
+		[NoWatch, NoTV, NoMac, iOS (14, 0)]
+		[Field ("kCVPixelBufferProResRAWKey_BlackLevel"), Advice ("This API is not available when using MacCatalyst.")]
```

Comment:
minor/style: having `[Advice]` and `[Field]` on the same line is more likely to be harder to review in the future
e.g. the advice might be be tru next year, if catalyst support is added, and someone could miss a typo in the field name (but intro should spot it)

They can be on different lines or you can combine the catalyst ones together, since they will be updated together.
```
[Unavailable (PlatformName.MacCatalyst), Advice ("This API is not available when using MacCatalyst.")]]
[Field ("kCVPixelBufferProResRAWKey_BlackLevel")]
```